packages/ui/src/styles/base/_typo.scss

```diff
@@ -1,27 +1,55 @@
+@mixin block-margin {
+  margin: 0 0 rem(20) 0;
```

Comment:
Shouldn't that also be em()?

Reply:
I am not sure for this one right now. I did not check how they separate paragraph.